server/handler/http_funtional_cluster_test.go

```diff
@@ -0,0 +1,737 @@
+//go:build all || functional
```

Comment:
there's a typo in the filename, it should be `http_functional_cluster_test.go`.

Reply:
done!